app/views/layouts/application.html.erb

```diff
@@ -9,6 +9,8 @@
   </head>
 
   <body>
+    <p class="notice"><%= notice %></p>
+    <p class="alert"><%= alert %></p>
```

Comment:
can be changed to flash helper http://stackoverflow.com/questions/12345065/rails-flash-message-helper